From a8ce13bbb17c9dc7516bf5481ecf0a6f759f28fd Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 1 Oct 2010 23:15:22 +0800
Subject: [PATCH 183/442] ENGR00132068 set mode for v4l2 capture open

gcc 4.4.4 toolchain requires 3 arguments for open
function with O_CREAT in second argument. Add
mode argument for capture open function to fix
compile error

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_still.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_still.c b/test/mxc_v4l2_test/mxc_v4l2_still.c
index 17e8a19..662b4b7 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_still.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_still.c
@@ -222,7 +222,7 @@ int v4l_capture_test(int fd_v4l)
         char *buf1, *buf2;
         char still_file[100] = "./still.yuv";
 
-        if ((fd_still = open(still_file, O_RDWR | O_CREAT | O_TRUNC)) < 0)
+        if ((fd_still = open(still_file, O_RDWR | O_CREAT | O_TRUNC, 0x0666)) < 0)
         {
                 printf("Unable to create y frame recording file\n");
                 return -1;
-- 
1.8.0

